src/main/scala/org/broadinstitute/dsde/workbench/sam/openam/JndiAccessPolicyDAO.scala

```diff
@@ -168,6 +168,19 @@ class JndiAccessPolicyDAO(protected val directoryConfig: DirectoryConfig)(implic
     }
   }
 
+  override def listAccessPoliciesWithEmail(resourceTypeName: ResourceTypeName, userId: WorkbenchUserId): Future[Set[ResourceIdAndPolicyNameWithEmail]] = {
+    listAccessPolicies(resourceTypeName, userId).flatMap { ripns =>
+      Future.traverse(ripns) { ripn => getPolicyEmail(resourceTypeName, ResourceIdAndPolicyName(ripn.resourceId, ripn.accessPolicyName)).map(email => ResourceIdAndPolicyNameWithEmail(ripn.resourceId, ripn.accessPolicyName, email)) }
```

Comment:
one extra query per managed group

alternatively i could re-write the `listAccessPolicies` query but then we don't take advantage of the `memberOf` attribute and the id/policy name being encoded in the dn